config/types.go

```diff
@@ -16,40 +16,34 @@ type Size int64
 
 // UnmarshalTOML parses the duration from the TOML config file
 func (d *Duration) UnmarshalTOML(b []byte) error {
-	var err error
-	b = bytes.Trim(b, `'`)
-
-	// see if we can directly convert it
-	dur, err := time.ParseDuration(string(b))
-	if err == nil {
-		*d = Duration(dur)
-		return nil
-	}
-
-	// Parse string duration, ie, "1s"
-	if uq, err := strconv.Unquote(string(b)); err == nil && len(uq) > 0 {
-		dur, err := time.ParseDuration(uq)
-		if err == nil {
-			*d = Duration(dur)
-			return nil
-		}
-	}
+	// convert to string
+	durStr := string(b)
 
+	// Value is a TOML number (e.g. 3, 10, 3.5)
 	// First try parsing as integer seconds
-	sI, err := strconv.ParseInt(string(b), 10, 64)
+	sI, err := strconv.ParseInt(durStr, 10, 64)
```

Comment:
What is someone inputs `"3"`? This `ParseInt` will fail but it will also fail in `ParseDuration`.

Reply:
yes because that is invalid input. See the test case for that option, it expects a failure.

Reply:
Oops your right, overlooked the test cases you added.